Automattic/newspack-plugin
# Fix/ga4 on amp

### All Submissions:






### Changes proposed in this Pull Request:



Adds GA4 support to AMP pages.

#### Questions 

- Is the ping to DoubleClick (`stats.g.doubleclick.net`) necessary? I'm assume it's analytics for GAM – cc @kmwilkerson 

### How to test the changes in this Pull Request:

1. Ensure a GA4 property is configured in Site Kit:

<img width="537" alt="image" src="https://user-images.githubusercontent.com/7383192/168995006-684de5dc-53f9-4e56-813e-fd468b84200a.png">

2. Load an AMP page in an incognito session, observe in your GA4 dashboard* (real-time view) that a visit has been recorded 

\* Search for the GA4 property in the [same old GA dashboard](https://analytics.google.com/analytics/web) 

### Other information:
